### ISSUE TITLE
Create CityCamp LinkedIn Group

### ISSUE THREAD

**Issue by ashumz** [ashumz/citycamp#43]
Do we know who has admin access/who created the LinkedIn group at https://www.linkedin.com/groups/CityCamp-Raleigh-3944736?  Would be nice to be able to rebrand with new logo and have access to reaching out to those members when the time comes

**Comment by ChrisTheDBA** [comment]
good idea - retire the old group since we don't know who the admins are.

**Comment by ashumz** [comment]
waiting on new logo to take over before creating this
